### DISCUSSION TITLE
We have a "channels" feature which allows you to assign a product to multiple channels, and each channel can represent a seller.

### DISCUSSION THREAD

**Discussion by loveyandex** [vendurehq/vendure#1022] [General]
you said month ago above title 
so we should able assign a user(seller) to a channel ...
are you sure this linking is possible?

we need new feuture : multi seller 
each seller can have itself products and orders 
just in one vendure server 
your platform is for single  seller

**Comment by loveyandex** [comment]
> Multi-Tenant (Marketplace) Support
Channels can also be used to implement a multi-tenant or marketplace application. In such a setup, each merchant would have their own dedicated Channel and would be granted permissions on that Channel only

where is code of channels to set a merchant(seller) to a channel and with own prdoducts

**Comment by loveyandex** [comment]
how can i define new product for a specific channel
or instead how can add that product to a spec channel?

there nk code in your document above told

**Reply by michaelbromley** [reply]
> how can i define new product for a specific channel

When creating a product with the `createProduct` mutation, the newly-created Product will be assigned to the current channel (as defined by the [`vendure-token` header](https://www.vendure.io/docs/storefront/shop-api-guide/#universal-parameters)).

> or instead how can add that product to a spec channel?

You can use the [assignProductsToChannel mutation](https://www.vendure.io/docs/graphql-api/admin/mutations/#assignproductstochannel)

**Reply by loveyandex** [reply]
@michaelbromley 
so what should we do about seller entities?
wallet and payment management after any order issued ?
i think we must write a plugin for this demand
and can i relate enteties together in plugins or modify a forien key of a core table by plugin?
if it is possible entirely modifing db we can solve multi vendor/seller problem

**Reply by michaelbromley** [reply]
The simplest way to model a Seller is to use the Administrator entity to represent a Seller, but created a Role limited to their specific channel.

Yes, you would write a plugin for your custom logic. See the [plugin examples docs](https://www.vendure.io/docs/plugins/plugin-examples/) for ideas on what can be done with plugins.

> wallet and payment management after any order issued ?

Not sure what the question is here. Can you elaborate?

**Reply by loveyandex** [reply]
> Not sure what the question is here. Can you elaborate?
should not have each seller revenue field for orders issued for ?
i will consider an entity like wallet for each seller in my plugin
or maybe a filed in adminstrator

**Reply by michaelbromley** [reply]
This is up to you, you can define whatever entities and business logic you need to support your workfow. But Vendure does not include any of that by default.

**Reply by lusicsmusics1** [reply]
@michaelbromley 
> you can define whatever entities and business logic you need to support your workfow.



is it possible vendure plugin modify a table and entity such as product and order table? beacuse new bissiness has its logic
